sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/IntervalBenchmark.scala

```diff
@@ -88,8 +78,6 @@ object IntervalBenchmark extends SqlBasedBenchmark {
     val intervalToTest = ListBuffer[String]()
 
     val benchmark = new Benchmark("cast strings to intervals", N, output = output)
-    addCase(benchmark, N, "string w/ interval", buildString(true, timeUnits))
-    addCase(benchmark, N, "string w/o interval", buildString(false, timeUnits))
```

Comment:
I believe this was a mistake. The first 2 cases do not cast string to interval and are very fast.

Since the first 2 cases simply test all units, it's already covered below.

Reply:
I have already told you the purpose of the 2 benchmarks: https://github.com/apache/spark/pull/26189#discussion_r337046377